apps/dashboard/src/components/welcome/ai-prompts/framework-prompts/nextjs-prompt.ts

```diff
@@ -14,8 +14,8 @@ export default function NotificationInbox({ subscriberId }: { subscriberId: stri
       subscriberId={subscriberId}
 
       // Backend configuration (for EU region use https://eu.api.novu.co and https://eu.ws.novu.co)
-      backendUrl=""
-      socketUrl=""
+      backendUrl=process.env.NOVU_BACKEND_URL
+      socketUrl=process.env.NOVU_SOCKET_URL
```

Comment:
@iampearceman why not just inlining those? Without the use of env variables? 


```suggestion
      backendUrl="https://eu.api.novu.co"
```